maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -223,6 +224,11 @@
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-maven-plugin</artifactId>
        <version>3.0.2</version>
+        <configuration>
+            <systemPropertyVariables>
+                <logging.config>src/main/resources/logback-dev-mode.xml</logging.config>
+            </systemPropertyVariables>
```

Comment:
`systemPropertyVariables` is how we pass `-D` properties through the spring-boot-maven-plugin

Reply:
And this config value gets overriden somewhere with `resources/logback-dev-mode.xml` when the app is deployed?

Reply:
This one certainly not because it is the one from Spring Boot and is not used in the docker image. But maybe the one we are passing to maven exec may be problematic. 

What I expect is that nothing will be passed to the jvm in docker unless explicitly configured in the docker plugin. But we will need to check. I will be surprised if the docker image is build taking into consideration the configurations that we pass to `mvn exec`. 

But let's confirm it first.

Reply:
Ok, I can confirm that none of the configs passed to `spring-boot-maven-plugin` or `exec-maven-plugin` are picked by `docker-maven-plugin`. 

The entry point in the image is: 

```
"Entrypoint": [
    "java",
    "-jar",
    "/maven/spring-fibonacci-action-1.0-SNAPSHOT.jar"
]
```

Which is exactly what we declared in `docker-maven-plugin`. So not hidden magic.